IDE/MonoDevelop/MonoDevelop.MonoGame/templates/Common/Program.cs

```diff
@@ -13,69 +13,69 @@
 
 namespace ${Namespace}
 {
-	#if __IOS__ || __TVOS__
-	[Register("AppDelegate")]
-	class Program : UIApplicationDelegate
-	#else
-	static class Program
-	#endif
+    #if __IOS__ || __TVOS__
+    [Register("AppDelegate")]
+    class Program : UIApplicationDelegate
+    #else
+    static class Program
+    #endif
     {
-        private static Game1 game;
-
-		internal static void RunGame() 
-		{
-			game = new Game1();
-			game.Run();
-		}
-			
+        internal static void RunGame() 
+        {
+            using (var game = new Game1())
+            {
+                game.Run();
```

Comment:
> On the side note, I think the whole Program.cs file is a mess, 

Yeah...  unfortunately the startup process for a bunch of platforms is weird and different from others.

We should open an issue on this specifically and discuss how to approach it.